File: abc/abc273/a.java
// https://atcoder.jp/contests/abc273/submissions/41532970

import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner stdIn = new Scanner(System.in);

    int n = stdIn.nextInt();

    int ans = 1;
    for(int i = 2; i <= n; i++) ans *= i;

    System.out.println(ans);
  }
}

File: abc/abc273/a.cs
// https://atcoder.jp/contests/abc273/submissions/46509459

using System;

public class A
{
  static void Main()
  {
    int n = int.Parse(Console.ReadLine());
    int ans = 1;

    for(int i = n; i > 0; i--) {
      ans *= i;
    }

    Console.Write(ans);
  }
}

